ajax/libs/ajv/package.json

```diff
@@ -22,7 +22,7 @@
     {
       "basePath": "dist",
       "files": [
-        "ajv.*"
+        "*.*"
```

Comment:
What about just `"*"` for all files not naming with a starting `.` (dot)? Or `"**/*"` for recursive including.